frontend/src/libs/components/app/styles.module.scss

```diff
@@ -1,4 +1,5 @@
 .container {
   flex-grow: 1;
   width: 100%;
+  margin-top: 105px;
```

Comment:
Let's move header height to variables and use it both for top-margin and for fixing header's height - then we'd be confident with this values